### DISCUSSION TITLE
SubNav is not automate close when mouse leave the Nav component

### DISCUSSION THREAD

**Discussion by giabaoye** [microsoft/fluentui#31028] [Q&A]
I use Nav component, it look like this:
[https://stackblitz.com/edit/vitejs-vite-jra1af?file=src%2FApp.tsx](url)
when the nav collapsed, I hover mouse on "Home", it will show the sub-menu, right? But when I move the mouse away Nav component, sub-menu stays still. Is that a bug? Any props that can do things like: onMouseLeave will make the sub-menu disappeared

**Comment by khmakoto** [comment]
Hey @baokun2115, I get a 404 error when I try to visit the link you pasted, can you double check if permissions are set correctly on that link?

![image](https://github.com/microsoft/fluentui/assets/7798177/b1f17381-6f74-49ff-a07b-55e3ef97c1f2)
